client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -11,10 +11,8 @@ import wrapWithClickOutside from 'react-click-outside';
  */
 import { transferStates } from 'state/automated-transfer/constants';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import {
-	getAutomatedTransferStatus,
-	isAutomatedTransferTransferring,
-} from 'state/automated-transfer/selectors';
+import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+import isAutomatedTransferActive from 'state/selectors/is-automated-transfer-active';
```

Comment:
Since we are exporting it from `state/selectors/index.js`, would it be better to use that for importing?
```javascript
import { isAutomatedTransferActive } from 'state/selectors';
```